models/Translation/Dao.php

```diff
@@ -168,6 +168,11 @@ public function getAvailableDomains(): array
     public function isAValidDomain(string $domain): bool
     {
         try {
+            $translationDomains = $this->model->getRegisteredDomains();
+            if (!in_arrayi($domain, $translationDomains)) {
```

Comment:
```suggestion
            if (!in_array($domain, $translationDomains)) {
```
@kingjia90 I think this suggestion was valid as you table names are case sensitive. why removed it?

Reply:
ah, i wasn't sure about that, i would expect devs to stick with snakecase but yes, it should be case sensitive afterall